### ISSUE TITLE
Further refinement on beelancing

### ISSUE THREAD

**Issue by Grath** [NillerMedDild/Enigmatica6#662]
**Modpack version**
0.3.2

**Issue**
In about 12 hours with a Tier 1 Apiary and a multiblock centrifuge, my two steel bees had produced 3 ingots. In the same time, two iron bees produced ~360 ingots. While iron is a much more fundamental resource, this makes steel bees almost pointless to breed and use due to the abysmally low output.

**Comment by NillerMedDild** [comment]
Yikes, that is awful. We're remaking the bees again for 0.3.3, would you mind letting us know what you think after you've tried that version?

**Comment by Grath** [comment]
Certainly! I'm very excited with the Botania bees I saw in some of the commits on here, if those are going to show up in 0.3.3 (albeit I'm a little sad about no more free nether stars from the Wither bees from 0.3.0 but that's an understandable balance thing.)

**Comment by NillerMedDild** [comment]
0.3.4 is out which has the new bees, would you mind having a look? :)

**Comment by Grath** [comment]
tl;dr Bees are currently good early-midgame base material production, plus convenient endgame supplemental production; Blaze bees have no flower and Obsidian bees use wither roses but aren't tagged with Wither trait so neither is usable; plus a couple nitpicks.

I had to rebuild my stock of bees, between attrition (stone block apiary appears to be deadly to terrible bee AI, I rebuilt it with glass walls/ceiling) and new types of bees. So far, bee production is looking adequate for early-midgame where they're easier to obtain to get good materials: More significant investment in my apiary - a tier 2 apiary and a tier 4 apiary, four multiblock Centrifuges - has resulted in 23 uranium in 25 minutes, but my 72-drill Industrial Foregoing laser drill feeding into a Mekanism 5x-ore processing line creates that much uranium in a matter of ~3 minutes. Bees seem like they're in a reasonable state for early-midgame resource production plus the alloy bees and the mob-bees/fundamental-blocks-bees are convenient for lazy folks like myself who got used to packs with Magical Crops letting us produce absolutely anything.

**Must-fix issues:**

- Obsidian bees require wither roses, but aren't tagged with Wither trait so the wither roses kill them. (I just opened [a feature request with Resourceful Bees](https://github.com/Resourceful-Bees/ResourcefulBees/issues/109) that they automatically configure Wither Rose-requiring bees to not die to wither roses to prevent immediately-suicidal-bee problems in packs; while this is a catch-it-in-QA type problem, it's still something that seems eminently fixable with a technical solution.)
- Blaze bees have no flower, so they appear to be unable to produce anything. I cheated in a blaze bee spawn egg (I could only find one in the nether and they're unbreedable so I just gave up and explored other options for blaze rods) to confirm in an apiary that previously successfully produced blaze rods.

Nitpick-y thoughts:

- Enderium bees may be a bit too productive; it takes 3600 ticks for a diamond bee to produce a 5% chance for diamonds, while it takes 2400 ticks for Enderium bees to produce 10% chance at Enderium ingots which are worth half a diamond each so Enderium bees are on average going to produce more diamonds than diamond bees are.
- Why are ghast and blaze bees unable to breed? Even Wither bees can be bred, at a high cost. With the number of biomes in the nether, I was only ever able to find one blaze bee.

**Comment by NillerMedDild** [comment]
> Blaze bees have no flower, so they appear to be unable to produce anything. I cheated in a blaze bee spawn egg (I could only find one in the nether and they're unbreedable so I just gave up and explored other options for blaze rods) to confirm in an apiary that previously successfully produced blaze rods.

Fixed in #770 

> 
    Enderium bees may be a bit too productive; it takes 3600 ticks for a diamond bee to produce a 5% chance for diamonds, while it takes 2400 ticks for Enderium bees to produce 10% chance at Enderium ingots which are worth half a diamond each so Enderium bees are on average going to produce more diamonds than diamond bees are.
    Why are ghast and blaze bees unable to breed? Even Wither bees can be bred, at a high cost. With the number of biomes in the nether, I was only ever able to find one blaze bee.

Will be fixed shortly, thanks for your thoughts :)
